java/pom.xml

```diff
@@ -41,6 +41,7 @@
     <arrow.vector.classifier></arrow.vector.classifier>
     <forkCount>2</forkCount>
     <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
+    <javac.version>9+181-r4173-1</javac.version>
```

Comment:
do we need this? should we not support anything java8 +?

Reply:
if java9+  is required for error prone - please note this will break all clients who are building arrow with java8..

can we make this an additional profile so that clients are not impacted..

Reply:
Not quite familiar with error prone and maven, just change pom according to the docs https://errorprone.info/docs/installation.
Can you give more details what should we do in the pom.xml? thanks